File: sort/programmers/biggestNumber/bella.py
def solution(numbers):
    answer=''
    numbers = map(str,numbers)
    nd=sorted(numbers,key=lambda x:x*3,reverse=True)
    for i in nd:
        answer = answer + i
    print(nd)
    if answer =='000':
        answer = '0'
    return answer

print(solution([15,1515]))
print(solution([0,0,0]))
print(solution([40,404]))

File: heap/programmers/scovilleScore/bella.java
import java.util.*;

class Solution {

    public int solution(int[] scoville, int K) {
        int answer = 1;
        //ArrayList<Integer> list = new ArrayList<>();
        PriorityQueue<Integer> pq = new PriorityQueue<>();
        for(int a : scoville){
            //list.add(a);
            pq.add(a);
        }

        int firstNum = 0;
        int secondNum = 0;
        int result = 0;
        while(true){
            if(pq.size() == 1){
                answer = -1;
                break;
            }
            //firstNum = Collections.min(list);
            //list.remove(Collections.min(list));
            //secondNum = Collections.min(list);
            //list.remove(Collections.min(list));
            firstNum = pq.poll();
            secondNum = pq.poll();
            result = firstNum + (secondNum * 2);
            pq.add(result);

            if(pq.peek() >= K)
                break;
            else
                answer++;
        }
        return answer;
    }
}